magellan/blocker/attr_equiv_blocker.py

```diff
@@ -150,48 +162,37 @@ def validate_block_attrs(self, ltable, rtable, l_block_attr, r_block_attr):
             r_block_attr = [r_block_attr]
         assert set(r_block_attr).issubset(rtable.columns) is True, 'Right block attribute is not in the right table'
 
-    def output_columns(self, l_key, r_key, col_names, l_output_attrs, r_output_attrs, l_output_prefix, r_output_prefix):
+    def output_columns(self, l_key, r_key, col_names, l_output_attrs,
```

Comment:
@sanjibkd  the utility functions should be made private (i.e., function name prefixed by _ )?